Src/Support/GoogleApis.Core/Apis/Requests/Parameters/ParameterUtils.cs

```diff
@@ -29,6 +30,9 @@ namespace Google.Apis.Requests.Parameters
     /// </summary>
     public static class ParameterUtils
     {
+        //TODO: Find out how to resolve the issue with setting up the Logger:
+        private static readonly ILogger Logger = ApplicationContext.Logger.ForType<ParameterUtils>();

```

Comment:
The issue here is that the class of ParameterUtils is static. If I attempt to create the logger as I've seen in some of the other classes, I get the following error: 

    Error	1	'Google.Apis.Requests.Parameters.ParameterUtils': static types cannot be used as 
    type arguments

Honestly, I've rarely used static classes so I've never run in to this, but some quick searches show that you [can't use static classes with generics by design of C#](http://stackoverflow.com/a/5858670/792032). @peleyal  had [mentioned](https://github.com/google/google-api-dotnet-client/pull/650#discussion_r49464086) using a static constructor, but that has no effect since the issue isn't how it's being set.